app/src/main/java/com/example/aurora/PersonalThemeActivity.java

```diff
@@ -44,31 +38,20 @@ protected void onCreate(Bundle savedInstanceState) {
 
     }
 
-    /**
-     * Sets up fragment managers
-     */
-    void setFM() {
-        FragmentManager fm = getSupportFragmentManager();
-       /* Fragment fragmentTheme = fm.findFragmentById(R.id.fragment_theme);
-        if (fragmentTheme == null) {
-            fragmentTheme = new ThemeFragment();
-            fm.beginTransaction()
-                    .add(R.id.fragment_theme, fragmentTheme)
-                    .commit();
-        }*/
+    @Override
+    protected void onPause() {
+        super.onPause();
+        (new Thread(new workerThread("2"))).start(); //THIS WILL STOP THE OCEAN LIGHTS
```

Comment:
2 - magic constant

Reply:
The constant 2 here used to send a message to the Raspberry pi. When the Raspberry pi receive a "2" is will stop playing the ocean theme.

Reply:
Cool. Extract the 2 in a constant with a self-explanatory name. e.g. STOP_PLAYING_OCEAN_THEME = 2.

Reply:
Okay, thank you very much.